src/common/modules/Notifications.js

```diff
@@ -16,20 +16,20 @@ const ICON = browser.runtime.getManifest().icons[32];
 export let SEND = true;
 
 /**
- * Show a notification.
+ * Show a notification. If the notification is optional, the user can disable it with an add-on setting. If it is required, the notification will always be shown. Use the latter notification type only sparsely, e.g. for user-initiated actions providing feedback to the user.
```

Comment:
```suggestion
 * Show a notification.
 *
 * If the notification is optional, the user can disable it with an add-on setting. If it is required, the notification will always be shown. Use the latter notification type only sparsely, e.g. for user-initiated actions providing feedback to the user.
```

Formatting and also what is the notification type/and you can use `@link`  (https://jsdoc.app/tags-inline-link.html) to link to params.